packages/bitcore-wallet-service/src/config.ts

```diff
@@ -1,188 +1,203 @@
-module.exports = {
-  basePath: '/bws/api',
-  disableLogs: false,
-  port: 3232,
+const { logger } = require('./lib/logger');
```

Comment:
Should use the `import` syntax instead of `require`. This package is kind of weird in that it mixes ES6 import syntax with CommonJS module.exports, but we should stick to the ES6 syntax where possible.

Reply:
Updated